data_models/environments/dev

```diff
@@ -0,0 +1,8 @@
+DJANGO_SECRET_KEY=!f3p74+hi5omz+)m((+8597&&01h-&bf(0#d1mwvak&clmv*%g
```

Comment:
Sure. I guess any string that will remind both users and developers immediately of its fakeness is good.